packages/git/src/index.mocked.test.ts

```diff
@@ -0,0 +1,61 @@
+import { execSync } from 'child_process'
+import { promises as fs } from 'fs'
+
+import { gitLastTaggedCommit, gitTag } from '.'
+
+jest.mock('monodeploy-logging')
+
+describe('monodeploy-git (mocked invariants)', () => {
+    let tempRepositoryRoot
+    async function setupTestRepository(): string {
+        const rootPath = await fs.mkdtemp('test-repository-')
```

Comment:
You can join this with os.tmpdir so it's not in the repo:

```suggestion
        const rootPath = await fs.mkdtemp(path.join(os.tmpdir(), 'test-repository-'))
```